sonar-db/src/main/java/org/sonar/db/permission/GroupPermissionMapper.java

```diff
@@ -28,13 +28,35 @@
 
 public interface GroupPermissionMapper {
 
+  /**
+   * @deprecated does not support organizations
+   */
+  @Deprecated
   int countGroups(Map<String, Object> parameters);
 
+  /**
+   * @deprecated does not support organizations
+   */
+  @Deprecated
   List<String> selectGroupNamesByPermissionQuery(@Param("query") PermissionQuery query, RowBounds rowBounds);
 
+  /**
+   * @deprecated does not support organizations
+   */
+  @Deprecated
   int countGroupsByPermissionQuery(@Param("query") PermissionQuery query);
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)